gp2gp-translator/src/test/resources/xml/AllergyIntolerance/allergy-structure-with-original-text-in-code.xml

```diff
@@ -29,9 +29,6 @@
                                     <translation code="416098002" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Allergy to drug"/>
                                     <translation code="14L..00" codeSystem="2.16.840.1.113883.2.1.6.2" displayName="H/O: drug allergy"/>
                                 </code>
-                                <value type="CD" codeSystem="1.3.6.1.4.1.17340.1.5" code="9006113"
-                                       displayName="Propranolol 10mg tablets (Almus Pharmaceuticals Ltd) @ 28 tablet(s)">
-                                </value>
```

Comment:
We've modified this test fixture, and I'm confused as to whether it's a valuable test at all anymore? Would we ever expect to see a `CompoundStatement` which looked like this?

It feels a little to me like we're creating a test fixture to match some behaviour our code has, as opposed to the other way round.

Reply:
Was NIAD-2621 the original ticket which prompted this behaviour/test? Perhaps we should go back to the `TPP` GP2GP XML extract as a source and see what it looked like.

Reply:
Whilst on the Medicus call, there was an example provided which had no value but had an originalText in the code section, so I would suspect this is a valid use case.